packages/augur-ui/src/modules/market/components/market-trade-history/market-trade-history.styles.less

```diff
@@ -11,11 +11,9 @@
   }
 
   > div {
-    display: flex;
-    flex-direction: column;
-    flex-grow: 1;
+    display: block;
     overflow: overlay;
-    max-height: 50vh;
+    max-height: calc(50vh + 53px);
```

Comment:
can you use one of these?
@size-50: 3.125rem; // 50px
@size-52: 3.25rem; // 52px